docs/developers/testing.rst

```diff
@@ -0,0 +1,110 @@
+==================
+Run MicroOVN tests
+==================
+
+MicroOVN has two types of tests, linter checks and functional tests and this
+page will show how to run them.
+
+Linter checks
+-------------
+
+Linting is currently applied only to the tests themselves, not the main
+codebase. The prerequisites for running linter are:
+
+* ``make``
+* ``shellcheck``
+
+You can install them with:
+
+.. code-block:: none
+
+   sudo apt install make shellcheck
```

Comment:
for consistency, as in the other how-to, let's mention these prerequisites inline and format the paragraph in the same way

Reply:
There's possibility that the list of dependencies will grow if/when we expand linting. If it's important to have paragraphs like these consistent, I'd rather change the other to a bullet list